jekyll/index.markdown

```diff
@@ -411,6 +412,13 @@ renaming can be triggered by right clicking the entity to rename or by pressing
 
 ![Rename demo](images/rename.gif)
 
+### Find references
+
+The find references request allows users to both see a panel with a list of references or jump to reference locations.
+Note that only constants are currently supported, but support for methods, instance variables and local variables is planned.
```

Comment:
This closes `#202` so we should remove that from the list (but we should probably create a new issue for tracking any remaining `find references` work)

Reply:
We don't remove complete items from the roadmap. If we want to start doing it, then maybe we can create a separate PR because many of those items are now complete.

I created new issues for the remaining references work and listed them there.